manifests/install.pp

```diff
@@ -57,6 +57,11 @@
             $apt_key_source = 'https://www.virtualbox.org/download/oracle_vbox.asc'
           }
         }
+        if $::lsbdistcodename == 'buster' {
```

Comment:
It looks a hack because the upstream project does not support buster.
At least, we have to explain that with a comment.

@voxpupuli/collaborators Other points of view about this hack are welcome.

And please use `$facts` to call facts.

Reply:
Is there a ticket on the upstream issue? If there is it should be mentioned/linked in the comment.